app/src/main/java/net/gsantner/markor/model/AppSettings.java

```diff
@@ -355,6 +351,22 @@ public void toggleFavouriteFile(File file) {
     private static final String PREF_PREFIX_AUTO_FORMAT = "PREF_PREFIX_AUTO_FORMAT";
     private static final String PREF_PREFIX_VIEW_SCROLL_X = "PREF_PREFIX_VIEW_SCROLL_X";
     private static final String PREF_PREFIX_VIEW_SCROLL_Y = "PREF_PREFIX_VIEW_SCROLL_Y";
+    private static final String PREF_PREFIX_TODO_DONE_NAME = "PREF_PREFIX_TODO_DONE_NAME";
+
+    public void setLastTodoDoneName(final String path, final String name) {
```

Comment:
setPerFileTodoTxtDoneFilepath ?

Reply:
We force the done file to be in the same folder as the todo.txt file, so it is a name and not a path. And the other per file functions are all setLast*. We can change this, but I think we should change them all if we change this one.

Reply:
Oh I thought you did some change to select the file from anywhere, didn't check that